rapid7_insightidr/komand_rapid7_insightidr/actions/query/action.py

```diff
@@ -0,0 +1,57 @@
+import komand
+from .schema import QueryInput, QueryOutput, Input, Output, Component
+# Custom imports below
+from komand.exceptions import PluginException
+from komand_rapid7_insightidr.util.endpoints import QueryLogs
+from komand_rapid7_insightidr.util.resource_helper import ResourceHelper
+import json
+import time
+
+
+class Query(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='query',
+            description=Component.DESCRIPTION,
+            input=QueryInput(),
+            output=QueryOutput())
+
+    def run(self, params={}):
+        time_now = int(time.time())
+        request = ResourceHelper(self.connection.session, self.logger)
+        request_params = {
```

Comment:
Can I get a code comment explaining why we need these?

Reply:
And to explain the significance with the values used

Reply:
Logs are returned only from last 30 days. This parameter are required by API. We can give this as an input or we can set this here. In my opinion there is better to get all logs instead creating input.

What comment should I add here?
This variable is request param and set `from` and `to` parameters which are send to API. With this API return data `from` some time and `to` some time. In this case it is: "7776000000" miliseconds ago. 

Maybe if I add some variables this will be more precise?

Reply:
That could help bu you need to add a comment with what you explained here. If I am working on this code a year from now I will have no clue where you got 7776000 or any of the other numbers or why they are there

Reply:
Sorry for the delay.
In my opinion writing comment is not a good solution :) From year from now this comment will be probably out of date. IMO it's better to analyze pure code instead of using possibly outdated comment :)
But for now I will add this comment:
"7776000 - is for three months from now. Is here because InsightDR keep logs for three months in hot storage"
Is it good enough?

Reply:
@r7-mgorny that works for now! Thanks!